modules/core/src/Auth/Source/AdminPassword.php

```diff
@@ -59,6 +59,11 @@ protected function login(string $username, string $password): array
             throw new Error\Error(Error\ErrorCodes::WRONGUSERPASS);
         }
 
+        $pwinfo = password_get_info($adminPassword);
+        if (!$pwinfo['algo']) {
```

Comment:
Let's make this a bit more specific `=== null`

Reply:
updated in 804b42bc